nemo/collections/llm/gpt/model/hyena.py

```diff
@@ -223,7 +223,7 @@ class HyenaConfig(TransformerConfig, io.IOMixin):
     use_short_conv_bias: bool = False
     # Use this if you want to turn FP8 on for the linear layer in the mixer only. When using this, do not set
     #  Fp8 in the mixed precision plugin.
-    vortex_style_fp8: bool = False
+    vortex_style_fp8: bool = True
```

Comment:
Can we leave this False by default? This setting is only a good default for inference...